andela/AndelaEatsUI
# #162045873 Start and End date Validation

#### What does this PR do?
Check start date and end date

#### Description of Task to be completed?
Validate that the start date is not greater than the end date

#### How should this be manually tested?
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git) and
-  cd into AndelaEatsUI and checkout to this branch `fix-startDate-notGreater-EndDate-162045873`
- run `npm run bundle` and `npm run dev`
- Log in as an `Admin`, navigate to `admin dashboard`, click on  `Menus`. Try to send a start date greater than end date
- run `npm test` to ensure all tests are passing

#### What are the relevant pivotal tracker stories?
- [#162045873](https://www.pivotaltracker.com/n/projects/2178749/stories/162045873)

#### Background Context
Nil

#### Screenshots (If Applicable)
<img width="1085" alt="screen shot 2018-11-19 at 5 43 39 pm" src="https://user-images.githubusercontent.com/21138774/48721812-ec29b680-ec22-11e8-865a-3861bc132f30.png">
